scripts/results/arrange-data.py

```diff
@@ -24,16 +24,46 @@
 
 import argparse
 import datetime
+import readline
 
-def trunc_datetime(line):
+

```

Comment:
add a **very brief** but **informative** description to _each_ function. Take your time and properly think about the best description of each function. I am not asking for a lengthy ambiguous description. 

Remember, if you cannot come up with a short function description it means either (1) the function is not implemented or designed properly -- so you do not even know its responsibility, or (2) you need to think more deeply for better wording.